src/store/card.js

```diff
@@ -58,6 +59,16 @@ export default {
 					}
 				}
 
+				if (unassigned) {
+					if (card.assignedUsers.length > 0) {
+						allUnassigned = false
+					}
+
+					if (!allUnassigned) {
```

Comment:
Why not just returning false directly then instead of adding the `allUnassigned` variable? This code looks way more complicated than it needs to be :wink: